src/test/java/hudson/plugins/tics/AbstractApiCallTest.java

```diff
@@ -0,0 +1,91 @@
+package hudson.plugins.tics;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import org.junit.Test;
+
+public class AbstractApiCallTest {
+
+    private class NoProxyTestCase {
+        public List<Pattern> noProxyPatterns;
```

Comment:
Make this `ImmutableList<Pattern> noProxyPatterns`

Reply:
Fixed in https://github.com/jenkinsci/tics-plugin/pull/28/commits/6139d5dc4e4d65e14756fd10a5e6f4192e51e731

Reply:
Fixed in https://github.com/jenkinsci/tics-plugin/pull/28/commits/6430159d83c9e01189a6f7b61a41749ac3668a6a